### PR TITLE
Updates to DEVELOPER.md

### PR DESCRIPTION
**Pull request by megahirt**
## Description

Some needed updates to the DEVELOPER readme to remove old sections on Protractor E2E tests and to add a section on IDE developer experience.

### Type of Change

- documentation

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have reviewed the title & description of this PR which I will use as the squashed PR commit message
- [x] I have enabled auto-merge (optional)

### CODE REVIEW

File: docs/DEVELOPER.md [hunk 1]
```diff
@@ -47,6 +55,7 @@ On Windows, the project should be opened with the [Remote - WSL](https://marketp
 3. Install [Node 16.14.0](https://nodejs.org/en/download/). We recommend using [nvm](https://github.com/nvm-sh/nvm#installation-and-update).
 4. Clone the repo: `git clone https://github.com/sillsdev/web-languageforge`.
    1. Windows users, be sure to clone the project to the WSL file system (to keep VS Code, Git and the file system in sync)
+5. Run `npm install` (required for git pre-commit hook with Prettier)
```
Comment (myieye) [inline docs/DEVELOPER.md:58]: 👍

File: docs/DEVELOPER.md [hunk 2]
```diff
@@ -7,34 +7,42 @@ Welcome! We're glad that you are interested in helping develop Language Forge.
...
 On Windows, the project should be opened with the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) VS Code extension, so that all commands and extensions run on WSL.
 
+### IDE Developer Experience (DX)
+
+While Docker is great way to encapsulate all of the dependencies, build tools and run-time environment of the application, IDEs usually require locally installed extensions, tools and runtimes in order to provide type checking, code hints, and debugging capabilities that make for a better developer experience. To this end, the following locally installed tools/dependencies may be installed for good DX:
+
+- PHP 7.3
+- Composer
+- Node and npm
+- .Net SDK
```
Comment (myieye) [inline docs/DEVELOPER.md:44]: .Net? Are you just anticipating merging the repos?

**Review by myieye — APPROVED** [review]
Looks good, as long as mentioning .NET was intentional.

File: docs/DEVELOPER.md (hunk 2 above)
Comment (megahirt) [reply docs/DEVELOPER.md:44]: The .Net SDK is actually referring to LFMerge development, which while a separate repo, is very much a part of Language Forge development.  It was intentional and non-specific, if only to hint towards a more detailed DX section.

**Pull request merged**